Do not build two test tools on Android. The Android libc is missing some of
the functions needed by those two programs.
Index: libxml2/libxml2-2.9.0/Makefile.am
===================================================================
--- libxml2.orig/libxml2-2.9.0/Makefile.am
+++ libxml2/libxml2-2.9.0/Makefile.am
@@ -12,8 +12,8 @@ AM_CFLAGS = $(THREAD_CFLAGS) $(Z_CFLAGS)
 
 noinst_PROGRAMS=testSchemas testRelax testSAX testHTML testXPath testURI \
                 testThreads testC14N testAutomata testRegexp \
-                testReader testapi testModule runtest runsuite testchar \
-		testdict runxmlconf testrecurse testlimits
+                testReader testapi testModule runsuite testchar \
+		testdict runxmlconf testlimits
 
 bin_PROGRAMS = xmllint xmlcatalog
 
